### DISCUSSION TITLE
Witamy w dyskusji nad Miarkownikiem ciągu WiFi

### DISCUSSION THREAD

**Discussion by krzychoooo** [krzychoooo/Miarkownik#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by krzychoooo** [comment]
**Tutaj jest miejsce na ogólną dyskusję, pytania, problemy z uruchomieniem etc...**

**Comment by piotrux** [comment]
Proszę o informację które rules używać, ponieważ są różnice w configach

**Reply by krzychoooo** [reply]
rules_2020_12_13.txt jest aktualny.
Poprzedni nie ma obsługi kończącego się paliwa

**Reply by piotrux** [reply]
Ok. Na ten moment testuję wszystko na sucho, czekam na update związany ze stroną pieca, widzę jeszcze kilka innych zmian, są one znaczące?

**Reply by krzychoooo** [reply]
Chodzi o zmiany po wgraniu configa?

**Reply by piotrux** [reply]
Chodzi mi o zmiany w plikach rules

Np 
TaskValueSet 2,1,[piec#error]*17+53+[piec#sumError]

Tutaj jest 53 a w nowszym 33

**Reply by krzychoooo** [reply]
A OK. To jest bardziej kosmetyka. Główna zmiana w rules_2020_12_13.txt to obsługa końca paliwa.

**Comment by piotrux** [comment]
Mam mały problem ale chyba nie do rozwiązania. Chciałem trochę przedłużyć ramię ze względu na to, że moja klapka potrzebuje około 10cm od 0 do pełnego otwarcia, niestety serwo nie daje rady, tzn ustawia kąt 100 po czym słychać, że walczy i opuszcza na chyba kąt 5 stopni, tego już nie wiem bo soft tego nie rejestruje, jest ciągle 100 w sofcie. Aktualnie zmieniłem kąt ze 100 na 120 w rules

**Reply by krzychoooo** [reply]
Można spróbować zwiększyć czas podtrzymania serwa
np linię
timerSet,1,5    // 5sek
na
timerSet,1,15    //15sek

Byle nie przegiąć z czasem odświeżania Termometru 40s ustawionym w devices. Chodzi też o to by niepotrzebnie nie grzać serwa i zasilacza.

**Reply by piotrux** [reply]
Ok zmieniłem na 10sek i działa ale czy nie zadaję serwa? Może jest takie podobne z większym momentem?

**Reply by krzychoooo** [reply]
Czas 10sek spokojnie może być. Teoretycznie powinno działać non stop. Ale jednak jest zamknięte w dodatkowej obudowie co utrudnia odprowadzenie ciepła.

**Comment by piotrux** [comment]
Chciałbym dodać opcję aby miarkownik po przekroczeniu np 2'C powyżej ustawionej zamykał klapkę na kąt 0, dodałem ifa:
if [Termometr_1#Temperature]>[piec#temp_set]+2
     TaskValueSet 2,1,0
     TaskValueSet 2,3,0
     TaskValueSet 4,1,100
  endif

Ale chyba się to gryzie z tym if który jest bez +2

Nie znam się, obczajam na chłopski rozum.

PS zauważyłem, że przy delta -0.5 klapka otwiera się na chwilę na 50... Dzieje się tak przy np tłumieniu pieca jak przekroczy żądaną temperaturę, klapka jest na 5 delta na 0, po czym wpada delta -0.5 a klapka leci na jakieś kilkanaście sekund na 50. Chodzi o to, że według mnie soft nie sprawdza czy jest przekroczona temperatura żądana i zachowuje się tak jakby piec wygasał, czyli chce to podtrzymać i zwiększa kąt

**Reply by krzychoooo** [reply]
>  Chodzi o to, że według mnie soft nie sprawdza czy jest przekroczona temperatura żądana i zachowuje się tak jakby piec wygasał, czyli chce to podtrzymać i zwiększa kąt

Soft sprawdza czy jest przekroczona temperatura. Ale jeśli spada szybko w dół to próbuje zapobiec mocnemu przeregulowaniu poniżej nastawy. Jest to element "D" regulatora PID. 
Czy mowa jest o wędzarni gdzie podgrzany czynnik to powietrze czy o zwykłym piecu gdzie nagrzewa się mnóstwo wody i żelastwa? 
Jeśli to wędzarnia to może załorze nowy wątek, bo tam będą inne problemy niż w zwykłym piecu. Porady dotyczące wędzarni niekoniecznie będą dobre dla użytkowników zwykłych pieców. Dla przyszłych czytających może to zrobić teraz. Bo tu na github nie bardzo widzę możliwość przenoszenia poszczególnych wypowiedzi do innych wątków.

**Reply by piotrux** [reply]
W porządku mój błąd... Wątek dotyczy pieca co

**Reply by krzychoooo** [reply]
OK. Założyłem nowy wątek _Wykorzystanie miarkownika w piecach wędzarniczych_ .
Chyba tu na github nie da się przenieść dyskusji z jednego wątku do drugiego. To nie forum. Jakoś trzeba sobie radzić. Myślę że z biegiem czasu będzie tu na tyle dużo wątków(tematów) że każdy się zorientuje gdzie czytać/pisać.

**Reply by krzychoooo** [reply]
Wiem że prowadzenie tutaj dyskusji w porównaniu z jakimś forum jest trochę siermiężne no ale lepiej tak niż zero wsparcia.

**Comment by piotrux** [comment]
Aktualny jest ten:
on System#Boot do
TaskValueSet 2,1,25
TaskValueSet 2,2,38
TaskValueSet 2,3,0
TaskValueSet 3,4,0
Servo,1,0,25
endon 

// if use Domoticz
//on Termometr_1#Temperature>80 do
//    SendToHTTP 192.168.8.111,8080,/json.htm?param=switchlight&type=command&idx=5&switchcmd=On  //kolejność parametrów istotna
//endon

on Termometr_1#Temperature do
  TaskValueSet 3,3,[Termometr_1#Temperature]-[piec2#oldTemp]
  if [Termometr_1#Temperature]<[piec#temp_set]
     TaskValueSet 2,4,[piec#temp_set]-[Termometr_1#Temperature]  //set error
     TaskValueSet 2,3,[piec#sumError]+[piec#error]-0.3
if [piec#sumError]>115
   TaskValueSet 2,3,115
endif
     TaskValueSet 2,1,([piec#error]*17+33+[piec#sumError])*[piec#error] 
  endif

  if [Termometr_1#Temperature]>[piec#temp_set] 
     TaskValueSet 2,4,[piec#temp_set]-[Termometr_1#Temperature]
     TaskValueSet 2,1,0
     TaskValueSet 2,3,0
  endif

  if [piec#alpha]<5
    TaskValueSet 2,1,0
  endif
  
  if [piec#alpha]>100
    TaskValueSet 2,1,100
  endif
  if [piec2#deltaTemp]<-0.5
    TaskValueSet 2,1,50      //set alpha
  endif
  if [piec2#deltaTemp]>1
    TaskValueSet 2,1,0      //set alpha
    TaskValueSet 2,3,-10       //set sumError
  endif

  if [piec2#stopWork]=1
    TaskValueSet 2,1,0
  endif

  if [piec#alpha]!=[piec2#oldAlpha]
    Servo,1,0,[piec#alpha]
    timerSet,1,10
  endif
  TaskValueSet 3,1,[piec#alpha]       //set piec2#oldAlpha
  TaskValueSet 3,2,[Termometr_1#Temperature]   //set piec2#oldTemp
endon

on setTemp do                        // http://<IP>/control?cmd=event,setTemp=55
  TaskValueSet 2,2,%eventvalue%
endon
on stopWorkEvent do  // zamykanie
  TaskValueSet 3,4,%eventvalue%
endon

On Rules#Timer=1 do  //When Timer1 expires, do
   servo,1,0,9001
endon

**Reply by krzychoooo** [reply]
Teraz nie mam czasu na analizę, Może w poniedziałek. ale nie podoba mi się na pierwszy rzut oka 
```
endif
endif
```

**Założyłem nowy wątek Wykorzystanie miarkownika w piecach wędzarniczych .**

**Comment by piotrux** [comment]
Ok odpaliłem domoticza ... i jestem trochę zielony. W miarkowniku podałem IP i port domoticza, dalej utknąłem

**Reply by krzychoooo** [reply]
Założyłem nowy wątek dla Domoticza bo pewnie będzie wiele dyskusji. Może pan skopiować jeszcze raz to pytanie tam.

**Comment by Karolnaz** [comment]
Witam,
Miarkownik czasami po ponownym włączeniu ustawia sobie temperaturę 38°
Czy to normalne?
Mogę tą domyślną wartość zmienić?

Można zwiększyć częstotliwość odczytów temperatury w aplikacji?

Czy czas nieosiągania temperatury mogę wydłużyć?
Pozdrawiam

**Reply by piotrux** [reply]
Hej, 38'C ustawia się z automatu, możesz to zmienić w rules linijka 
TaskValueSet 2,2,38 - zmieniasz 38 na to co chcesz

Odczyty zmieniasz klikając na edit przy termometrze na dole masz interwał domyślnie chyba 40 sek. ale nie ma co przesadzać z częstotliwością

Co do ostatniego pytania nie wiem ale pewnie się da, używam rules bez wygaszania

**Reply by krzychoooo** [reply]
> Miarkownik czasami po ponownym włączeniu ustawia sobie temperaturę 38°
> Czy to normalne?
> Mogę tą domyślną wartość zmienić?

Można to zmienić. W Rules trzecia linijka ustawia temperaturę po resecie.
`TaskValueSet 2,2,38`
38 to właśnie temperatura.

> Można zwiększyć częstotliwość odczytów temperatury w aplikacji?

Można by było ale niema sensu generować ruchu w sieci ponieważ miarkownik odczytuje temperaturę co 40s.


> Czy czas nieosiągania temperatury mogę wydłużyć?

O który czas chodzi? Ten po naciśnięciu start w aplikacji czy w ogóle już podczas pracy?

**Reply by krzychoooo** [reply]
> Odczyty zmieniasz klikając na edit przy termometrze na dole masz interwał domyślnie chyba 40 sek. ale nie ma co przesadzać z częstotliwością

Dokładnie tak jak pisze piotrux to może nawet rozchwiać działanie regulatora. Te 40s zostało dobrane po wielu próbach na piecu. Więc ostrożnie z tym parametrem.

**Reply by Karolnaz** [reply]
Myślałem że te 40s to czas pomiędzy odczytami tylko przez aplikację.
Czyli regulacja opiera się o takie próbkowanie?

Z czasem nieosiągania temperatury miałem na myśli to odliczanie paliwa, bo widzę, że te 15minut może nie wystarczyć.

Mam też problemy z komunikacją. Router widzi miarkownik, ale logowanie się przez przeglądarkę i aplikację jest losowo skuteczne. Może router musi mieć konkretne ustawienia?

**Reply by krzychoooo** [reply]
> Myślałem że te 40s to czas pomiędzy odczytami tylko przez aplikację.
> Czyli regulacja opiera się o takie próbkowanie?

Tak.

> Z czasem nieosiągania temperatury miałem na myśli to odliczanie paliwa, bo widzę, że te 15minut może nie wystarczyć.

Przedostatnia linijka w Rules
`timerSet,2,20`
20 oznacza że Timer2 co 20 sekund zmniejsza paliwo o 2%. Można to zmieniać. np
`timerSet,2,40`
będzie zmniejszane co 40s

> Mam też problemy z komunikacją.

Na cyferblacie jest wyświetlany poziom RSSI w dBm. jaki on jest?

**Reply by Karolnaz** [reply]
Wskaźnik rssi się nie pojawiał, po kilku sek od włączenia aplikacji, znacznik nastawy ustawiał się na zero i komunikat o błędzie połączenia.Zdążyłem na nowo połączyć miarkownik z routerem (po raz drugi) i teraz śmiga. Wydaje mi się, że to po wyłączeniu z sieci na noc taki problem.

Dziękuję za odpowiedzi.

Mam jeszcze pytanie.
Co wpływa na resetowanie wskaźnika paliwa? Osiągnięcie temperatury?

**Reply by krzychoooo** [reply]
> Co wpływa na resetowanie wskaźnika paliwa? Osiągnięcie temperatury?

Tak.

**Reply by Karolnaz** [reply]
Jeżeli dobrze pamiętam to w sterownikach CO stosowane są różne parametry "czas nieosiągania temperatury". Dla rozpalania jest on znacznie dłuższy.

**Comment by Karolnaz** [comment]
"FAQ
3.Czy jest "wygaszanie" ?
Tak, jeśli przez 8 minut nie osiągnie temperatury nastawionej to zamyka klapkę. Po uruchomieniu czeka 40 minut."
Czy to obecnie tak funkcjonuje?

**Reply by Karolnaz** [reply]
Czy maksymalny kąt alpha podczas pracy wynosi 100°?

**Reply by piotrux** [reply]
Ustawiony w rules maksymalny kąt to 100°

**Reply by piotrux** [reply]
Z tego co wiem wygaszanie działa, ja nie używam

**Reply by Karolnaz** [reply]
Do wygaszania przydał by się ustawiany próg temp. Zejściu poniżej tej temperatury skutkowałoby zamknięciem klapy.

**Reply by piotrux** [reply]
Teraz działa to moim zdaniem lepiej tzn jeżeli temp nie rośnie przez 8min to znaczy, że nie ma paliwa i gasi piec.

**Reply by Karolnaz** [reply]
Zależy od pieca i paliwa. Przy dużym piecu na drewno i dużym odbiorze zdarza się, że zostaje jedno polano, albo więcej ale stoją na palenisku. Wiadomo że nie osiągnie się mocy i zadanej temperatury, ale będzie się to palić jeszcze przez godzinę. Zamknięcie klapy w tym wypadku nie jest lepsze, zwłaszcza w nocy.

**Reply by Karolnaz** [reply]
Dla uściślenia dopytam ile jest warunków czasowych kończących pracę? 
"Przedostatnia linijka w Rules
timerSet,2,20"
Coś jeszcze liczy czas do wyłączenia?

**Reply by krzychoooo** [reply]
Tak dokładnie to te linie odliczają czas:
```
On Rules#Timer=2 do
  TaskValueSet 4,1,[piec3#timeToOFF]-2   // zmniejszanie o2%
    if [piec3#timeToOFF]<0
      TaskValueSet 3,4,1   //fuel end -> stopWork blokada palenia
      TaskValueSet 4,1,0
    endif
  timerSet,2,20     // za ile sekund ponowne wyzwolenie timera
endon
```

**Reply by krzychoooo** [reply]
Ja na noc obniżam temperaturę, a właściwie nie ja tylko Domoticz. Tak aby się tliło. Bez problemu mogłem ustawić np 30C i nie wygasał (na zewnątrz było koło 12).
W styczniu pojawi się nowa wersja softu że będzie można ustawić 4 pary (czas, temperatura) do ustawienia. Pisałem już o tym
Ten miarkownik wykorzystywany jest czasem do wędzarni więc ustawienie na sztywno temp przy której ma się wyłączyć było by znowu inne niż w piecu CO. 
Ale to nowy projekt a więc dobrze że dyskusja trwa. Na szczęście soft da się zmieniać.

**Reply by piotrux** [reply]
Mogę zapytać jak Panu domoticz ustawia temperaturę? Chciałem coś wymyślić aby pilnował mi temp w pokoju dzięki netatmo ale nie jestem biegły w domoticzu to moja pierwsza styczność. Aktualnie mam odczyty temp miarkownika, kąta klapki, termostat którym steruje miarkownik i wszystkie czujki netatmo

**Reply by krzychoooo** [reply]
Tak jak w filmie "Integracja miarkownika z systemem Domoticz".
W zakładce Użytkowe klikamy **Plany**
![Screenshot_2020-12-31 Dom](https://user-images.githubusercontent.com/7173366/103421014-77176200-4b9a-11eb-9261-755b0cca67db.png)

i tam możemy ustawiać:
![Screenshot_2020-12-31 Dom-plany](https://user-images.githubusercontent.com/7173366/103421032-8c8c8c00-4b9a-11eb-8ed9-a5cbb08a654f.png)

**Reply by krzychoooo** [reply]
Jeśli to ma działać w zależności od czujek temperatury to trzeba napisać skrypt w "**Konfiguracja->więcej opcji-> Zdarzenia**"

**Comment by Karolnaz** [comment]
Zapytam z ciekawości.
Jakie mają Panowie piece (paliwo, moc), sterowniki np pompy CO, oraz ogrzewaną powierzchnię?

**Reply by krzychoooo** [reply]
Piec mam 20kW. Jeśli na zewnątrz jest powyżej 8C to palę drewnem dębowym i korzystam z miarkownika. 
Jeśli poniżej to ekogroszek z podajnikiem. Pewnie drewnem też by dało radę ale nie chce mi się tyle latać do kotłowni. 
Przy średniej 3C na zewnątrz starcza mi 4 do 5 worków25kG na 7dni.
Izolacja domu na 10cm. 
Dużo kaloryferów podwójnych. 
Na kaloryferach "wyciągacze ciepła WiFi" :-)
Myślę że kluczowe to jest ocieplenie oraz powierzchnia kaloryferów. Przy dużej powierzchni kaloryferów można obniżać temperaturę na piecu tym samym temperaturę spalin idących w komin.

**Reply by Karolnaz** [reply]
> Myślę że kluczowe to jest ocieplenie oraz powierzchnia kaloryferów. Przy dużej powierzchni kaloryferów można obniżać temperaturę na piecu tym samym temperaturę spalin idących w komin.

U moich teściów prymitywny piec "ma" 8kW, a grzejniki żeliwne  20kW. Efektem jest praca w zakresie 28-35°C.
Poprzedni piec wytrzymał ok 10lat i przerdzewiał na sito. Piec za 1tys zł

**Reply by Karolnaz** [reply]
>Na kaloryferach "wyciągacze ciepła WiFi" :-)

Co to takiego?

**Reply by krzychoooo** [reply]
Ciche wentylatorki plus odpowiednio sterowane przetwornice do nich plus czujnik temperatury no i plus WiFi. 
Na kaloryfer 1,2m około 10szt.
Jeszcze wszystko w testach i prototypach. Wyniki są obiecujące. Może końcem lutego do sprzedaży.

**Comment by Karolnaz** [comment]
Jaki jest optymalna pozycja dźwigni miarkownika?
0° w pionie do dołu?

**Reply by krzychoooo** [reply]
dla 0° klapa ma być zamknięta.

**Reply by Karolnaz** [reply]
Chodzi mi o pozycję dźwigni miarkownika przy 0°

**Reply by krzychoooo** [reply]
Chyba nie bardzo rozumiem pytania.
Czy chodzi o zero stopni kątowych czy zero stopni Celsius? Czy jak w aplikacji naciśniemy zero stopni?

**Reply by Karolnaz** [reply]
Montuję miarkownik na ścianie pieca, ustawiam w aplikacji kąt 0° i muszę przymocować dźwignię miarkownika.
Zamontowałem w dół, ale zastanawiam się czy to optymalne ustawienie

**Reply by krzychoooo** [reply]
Tak to optymalne.
A tak dokładniej to dla 0 linia dźwigienki i elastycznego cięgna powinny stanowić prostą. Na jednej linii.

**Reply by Karolnaz** [reply]
Dziekuję

**Comment by AndyBoom** [comment]
Proponuję dołożenie do aplikacji przycisku ROZPALANIE, który wykorzystywałby dopisany fragment programu utrzymujący otwartą klapkę aż do osiągnięcia na kotle zadanej w aplikacji temperatury.

**Reply by piotrux** [reply]
> Proponuję dołożenie do aplikacji przycisku ROZPALANIE, który wykorzystywałby dopisany fragment programu utrzymujący otwartą klapkę aż do osiągnięcia na kotle zadanej w aplikacji temperatury.

Tak teraz jest

**Reply by AndyBoom** [reply]
Musiało mi to umknąć. W której wersji?

**Reply by piotrux** [reply]
Tzn miarkownik po uruchomieniu ustawia klapkę na max kąt, kąt ten jest maksymalny do momentu uzyskania rządnej temperatury. Wyjątkiem jest zbyt szybkie rozpalanie, kiedy to miarkownik przymyka klapkę aby proces rozpalenia nie nastąpił zbyt gwałtownie.

**Reply by AndyBoom** [reply]
Rozumiem, że to w którymś Rules. W którym? Mam coś podmienić w moim miarkowniku?

**Reply by krzychoooo** [reply]
w rules zamienić to:
```
  if [piec2#deltaTemp]<-0.2
    TaskValueSet 2,1,50      //set alpha
  endif
  if [piec2#deltaTemp]>0.2
    TaskValueSet 2,1,5      //set alpha
    TaskValueSet 2,3,-10       //set sumError
  endif
```
na to :
 ```
  if [piec2#deltaTemp]<-1
    TaskValueSet 2,1,25      //set alpha
  endif
  if [piec2#deltaTemp]>0.5
    TaskValueSet 2,1,5      //set alpha
    TaskValueSet 2,3,-10       //set sumError
  endif
```

**Reply by AndyBoom** [reply]
Mam spory problem z miarkownikiem. Nie reaguje na komendy przesyłane z aplikacji. W aplikacji wygląda na to, że wszystko działa ale w rzeczywistości nic się nie dzieje. Mimo połączenia i zaczytywania temperatury na piecu miarkownik nie reaguje nawet na komendę 0 i 180 stopni. Zdarza mu się zamknąć na chwilę klapkę (normalny ruch serwa) żeby za chwilę ją otworzyć (bardzo powolne otwieranie). Temperatura na piecu mogłaby rosnąć do maxa gdyby nie ręczne wypinanie klapki i jej zamykanie. Nie wiem czy uda się reanimować miarkownik. Na pewno potrzebuję POMOCY!

**Comment by piotrux** [comment]
Proszę o krótki opis zmian w rules rules_2021_02_17.txt ponieważ widzę, że jest mocno przebudowany

**Reply by AndyBoom** [reply]
Przyłączam się.
Ja chętnie zapoznałbym się z opisem znaczenia tych:
TaskValueSet 2,1,25
TaskValueSet 2,2,38
TaskValueSet 2,3,0
TaskValueSet 3,4,0
TaskValueSet 4,1,100 itp.
Część rozszyfrowałem, ale nie ukrywam, że zmienianie trzeciej wartości i obserwowanie jak wpłynie to na układ jest strasznie czasochłonne.

**Reply by krzychoooo** [reply]
W zakładce devices mamy urządzenia.
Każde urządzenie to numer tasku.
W ramach tasku możemy mieć 4 zmienne
I teraz
TaskValueSet 2,1,25      ustaw w tasku 2 zmienną nr 1 na 25
TaskValueSet 2,2,38      ustaw w tasku 2 zmienną nr 2 na 38
itd...

**Reply by krzychoooo** [reply]
W katalogu tutaj na github w katalogu rules jest nowa aktualizacja 2021_02_17

**Reply by krzychoooo** [reply]
> Proszę o krótki opis zmian w rules rules_2021_02_17.txt ponieważ widzę, że jest mocno przebudowany

Wykrywanie zbyt szybkiego wzrostu/spadku temperatury ograniczono do |error|<3    część D z PID
"Uzupełnienie " paliwa następuje jeśli error < 3
Zmiana wzoru wychylenia klapki na:
 alfa = error*35 + 10 + zsumowane błędy

Jeśli chodzi o ten wzór to jak ktoś ma zacięcie eksperymentatora to można potestować np. z parametrem k=35

error to:  temperatura nastawiona - temperatura odczytana

**Reply by krzychoooo** [reply]
Właśnie co do pisania rules to jest to mega niewygodne dlatego powstał film https://www.youtube.com/watch?v=tV3wzYl6GP4

**Reply by AndyBoom** [reply]
Dokładnie o takie informacje mi chodziło. Dziękuję!

**Comment by piotrux** [comment]
Dotyczy hardware: Co to za rezystor w wersji V2 na serwie
![Bez tytułu](https://user-images.githubusercontent.com/76114450/109147068-dfcb2700-7764-11eb-8824-a30881f014b9.jpg)
?

**Reply by krzychoooo** [reply]
Już odpisywałem na YouTube

**Comment by Potentator** [comment]
Czy moge poprosić o zrzuty ekranu odnośnie miarkownika devices i rules? Coś mi się rozjechał config i rules i od wczoraj nie działa mi miarkownik :(

**Reply by AndyBoom** [reply]
Czy udało się naprawić miarkownik? Mi niestety też padł

**Comment by piotrux** [comment]
Tak ale już nie używam tego rozwiązania od dość dawna (gaz aktualnie) więc
nie pamiętam co było przyczyną niestety
Pozdrawiam

czw., 29 gru 2022, 14:28 użytkownik AndyBoom ***@***.***>
napisał:

**Reply by AndyBoom** [reply]
A może mógłbym odkupić ten miarkownik?

**Comment by piotrux** [comment]
Może serwo padło? Trochę boję się pozbywać swojego miarkownika bo pewnie
już takiego nie dostanę a nie wiadomo jak będzie czy nie trzeba będzie
wrócić do kopciucha. Mam serwo sprawne mogę ci wysłać do testów

niedz., 1 sty 2023, 22:04 użytkownik AndyBoom ***@***.***>
napisał:

**Reply by AndyBoom** [reply]
Dziękuję za propozycję. Niestety mój miarkownik wyciął mi numer. Mimo tego, że po postawieniu go od nowa działał kilka dni bez problemów pewnego dnia pod moją nieobecność nie zamknął mi pieca i doprowadził do katastrofy. Zagotował wodę w całej instalacji wprowadzając piec w takie rejony temperaturowe, że rozerwało mi luty w odległości około 1,5 metra od pieca, nieużywany wentylator, który był zamontowany na kotle stopił się w grudę, mieszaninę metalu i spalonego plastiku a (na szczęście) niewielkie ilości sadzy w kominie zapaliły się. Było groźnie. Miarkownik odpadł od pieca (prawdopodobnie to spowodowało zamknięcie klapki) i chyba już nie wróci. Cieszę się, że mimo wszystko mam gdzie mieszkać...

**Reply by timothy1989** [reply]
Serwo zastosowane w tym projekcie nie nadaje się do podnoszenia i opuszczania klapki. Tranzystory zastosowane do sterowania silniczkiem są za słabe i pomimo zastosowania opornika wpiętego szeregowo lubią się przepalać. Sam wymieniałem serwo już 3 razy, ale po zastosowaniu SG-5010 od ponad roku mam spokój. Trzeba tylko wykasować z kodu komendy odpowiadające za wyłączenie serwa, czyli np. servo,1,0,9001 gdyż serwo i tak podtrzymuje zadaną pozycję. Kod przerobiłem pod własny projekt PCB, dodałem też przycisk fizyczny od zamykania klapy i podtrzymanie w razie zaniku zasilania z sieci.

**Comment by Potentator** [comment]
Mi działa 6 rok i jest ok, nic nie zmieniałem. 
     
      
       
        Dnia 09 marca 2024 13:04 timothy1989 ***@***.***&gt; napisał(a):
       
    
       
         
 Serwo zastosowane w tym projekcie nie nadaje się do podnoszenia i opuszczania klapki. Tranzystory zastosowane do sterowania silniczkiem są za słabe i pomimo zastosowania opornika wpiętego szeregowo lubią się przepalać. Sam wymieniałem serwo już 3 razy, ale po zastosowaniu SG-5010 od ponad roku mam spokój. Trzeba tylko wykasować z kodu komendy odpowiadające za wyłączenie serwa, czyli np. servo,1,0,9001 gdyż serwo i tak podtrzymuje zadaną pozycję.
